cadence/contracts/KittyItemsMarket.cdc

```diff
@@ -54,7 +60,7 @@ pub contract KittyItemsMarket {
     //
     pub resource interface SaleOfferPublicView {
         pub var saleCompleted: Bool
-        pub let saleItemID: UInt64
+        pub let saleItemId: UInt64
```

Comment:
The rest of these contracts used caps (`ID`) instead of `Id`, and I think that's becoming a de-facto standard in Cadence. I know that `Id` is standard in JS, but I'd recommend keeping this as-is in Cadence.

Reply:
Ok, wasn't aware this was a Cadence thing. Will revert.